packages/module/patternfly-docs/content/examples/PropertiesSidePanel.md

```diff
@@ -26,44 +26,42 @@ import OkIcon from '@patternfly/react-icons/dist/esm/icons/ok-icon';
 import ExternalLinkAltIcon from '@patternfly/react-icons/dist/esm/icons/external-link-alt-icon';
 import GlobeIcon from '@patternfly/react-icons/dist/esm/icons/globe-icon';
 
-<div style={{ display: 'inline-block', padding: '15px', border: '1px solid grey' }}>
```

Comment:
Took this wrapper off just so it's clear what style is coming from `<PropertiesSidePanel>` and what's just in the example code.